projects/observability/src/shared/components/gauge/gauge.component.ts

```diff
@@ -197,15 +199,13 @@ interface GaugeSvgRendererData {
   data?: GaugeData;
 }
 
-interface GaugeData {
+interface GaugeData extends GaugeInputData {
   valueArc: string;
-  value: number;
-  maxValue: number;
-  threshold: GaugeThreshold;
 }
 
 interface GaugeInputData {
   value: number;
   maxValue: number;
-  threshold: GaugeThreshold;
+  label: string;
+  color: Color | string;
```

Comment:
can we force only Color here? what non color string are we expecting here?

Reply:
There is no problem to force Color here, but GaugeThreshold could be Color or string, so I think that the better idea would be to maintain the same type in the rest of the interfaces. If you think that we can lose the functionality previously developed I can remove it without major issues.

Reply:
I suspect it would bubble up and be a pain - the gauge itself, less of a problem, but this is used in a widget where colors are coming from json. They're likely `Color`s (which is just our enum of predefined color hexes), but the type safety isn't there since they're coming from unknown strings. Since we use them in the exact same way anyway (this accepting a string just means it can take a css-compatible color string), I'm guessing that's where the current union comes from.

Reply:
I will keep it as it currently is, thanks for the clarification